### PR TITLE
Fix ::set-output deprecation messages

### PR DESCRIPTION
**Pull request by andros21**

### CODE REVIEW

File: scan-apk/action.yaml [hunk 1]
```diff
@@ -70,7 +70,7 @@ runs:
         tag: localhost:1234/apk-scan
 
     - id: grype-scan
-      uses: anchore/scan-action@ecfd0e98932e57ea8f68f29c4f418fc41a8194db
+      uses: anchore/scan-action@24fd7c9060f3c96848dd1929fac8d796fb5ae4b4
```
Comment (inteon) [inline scan-apk/action.yaml:73]: This change seems somewhat unrelated to the rest of the PR?

Comment (andros21) [reply scan-apk/action.yaml:73]: Yes, it seems unrelated but without this minor bump [`v3.2.5`](https://github.com/anchore/scan-action/releases/tag/v3.2.5) -> [`v3.3.6`](https://github.com/anchore/scan-action/releases/tag/v3.3.6) not all `::set-ouput` warnings go away, because used inside actual pinned version of `anchore/scan-action`. Maybe a different version can be selected, I have chosen this cause it's the last one released and surely in there `::set-output` warnings are fixed

Comment (cpanato) [inline scan-apk/action.yaml:73]: can you add the version tag that this git hash is pointing to? will make easier for the next time

**Review by cpanato — CHANGES_REQUESTED** [review]
thanks for the updates
a small nit

File: scan-apk/action.yaml (hunk 1 above)
Comment (andros21) [reply scan-apk/action.yaml]: Sure, added

**Comment by inteon** [comment]
@cpanato The PR now seems to be ready.

**Review by cpanato — APPROVED** [review]

File: apko-publish/action.yaml [hunk 2]
```diff
@@ -135,10 +135,12 @@ runs:
         --network host \
         -v $PWD:${{ github.workspace }} \
         -v /tmp:/tmp \
+        -v $GITHUB_OUTPUT:$GITHUB_OUTPUT \
```
Comment (cpanato) [inline apko-publish/action.yaml:138]: looking at it again, is this required?

File: apko-publish/action.yaml [hunk 3]
```diff
@@ -135,10 +135,12 @@ runs:
...
         --workdir ${{ github.workspace }} \
         -e "GITHUB_ACTOR=${{ inputs.repository_owner }}" \
         -e "GITHUB_TOKEN=${{ inputs.token }}" \
         -e "REPOSITORY=${{ inputs.repository }}" \
+        -e "GITHUB_OUTPUT=$GITHUB_OUTPUT" \
```
Comment (cpanato) [inline apko-publish/action.yaml:143]: and this?

Comment (andros21) [reply apko-publish/action.yaml:143]: Not strictly required, both are required if digest output (line 180) is done inside the docker run command. An alternative way could be something like this https://github.com/andros21/actions/pull/4/commits/0024871c27dd41f5ba3ee4d3cc83d3a2c9ec5718. If ok, I can update it

Comment (cpanato) [reply apko-publish/action.yaml:143]: i think you can remove this

**Review by cpanato — CHANGES_REQUESTED** [review]

**Comment by cpanato** [comment]
I did a rebase to update the branch and seems all were gone
